rholang/examples/vault_demo/demo_script.md

```diff
@@ -2,7 +2,6 @@
 
     #execute from the `examples` directory
     . keys.env
-    ./propose.sh $GENESIS_PRV vault_demo/0.create_genesis_vault.rho "-e s/%REV_ADDR/$GENESIS_REV/"
```

Comment:
This script should now cover how to run the node to have a vault with funds accessible